packages/profile-sync-controller/CHANGELOG.md

```diff
@@ -7,6 +7,40 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.0]
+
+### Added
+
+- Add network synchronisation logic ([#4694](https://github.com/MetaMask/core/pull/4694), [#4687](https://github.com/MetaMask/core/pull/4687), [#4685](https://github.com/MetaMask/core/pull/4685), [#4684](https://github.com/MetaMask/core/pull/4684))
+- Add a `canSync` check for account synchronisation ([#4690](https://github.com/MetaMask/core/pull/4690))
+
+### Changed
+
+- Bump `@metamask/snaps-sdk` from `^6.1.1` to `^6.5.0` ([#4689](https://github.com/MetaMask/core/pull/4689))
+- Bump `@metamask/snaps-utils` from `^7.8.1` to `^8.1.1` ([#4689](https://github.com/MetaMask/core/pull/4689))
+- Bump peer dependency `@metamask/snaps-controllers` from `^9.3.0` to `^9.7.0` ([#4689](https://github.com/MetaMask/core/pull/4689))
+
+### Removed
+
+- **BREAKING:** Remove `getAccountByAddress` action ([#4693](https://github.com/MetaMask/core/pull/4693))
```

Comment:
```suggestion
- Remove `getAccountByAddress` action ([#4693](https://github.com/MetaMask/core/pull/4693))
```

It's not really a breaking change since nothing depended on it before.
Apart from that, changelog LGTM so thanks a ton!

Reply:
It's technically a breaking change, since it was part of the public API, but no longer is. I don't think it makes much of a difference in this case since the controller is on version `<1.0.0`, meaning that we can introduce breaking changes in minor version bumps regardless.